index.js

```diff
@@ -144,7 +144,6 @@ const runChecks = async (context, checkEvent) => {
           case constants.periodicCheck:

```

Comment:
Add `/* istanbul ignore next */` just before https://github.com/oppia/oppiabot/blob/65f1717d62f4f3358db40909e3c4228398ee14c1/lib/periodicChecks.js#L151
so it doesn't get counted for code  coverage

Reply:
Also add `/* istanbul ignore next */` to
https://github.com/oppia/oppiabot/blob/65f1717d62f4f3358db40909e3c4228398ee14c1/lib/utils.js#L364

and 

https://github.com/oppia/oppiabot/blob/65f1717d62f4f3358db40909e3c4228398ee14c1/lib/utils.js#L406